### DISCUSSION TITLE
changed pressure advanced after update...

### DISCUSSION THREAD

**Discussion by Mario-vminion** [OrcaSlicer/OrcaSlicer#851] [Q&A]
hi i did a update from the actions section , and could it be that there is something changed with the setting pressure advanced that it not give the value what you put with the filament..

i have printed a thing after the update and it looks that the pressure advanced setting is not good anymore and before i had no problem...

**Comment by gasprints** [comment]
I also noticed this.  I slowed down the max volumetric speed and it helped a bit on my P1P, but I haven't been able to run proper PA and FR tests again.

**Reply by Mario-vminion** [reply]
i print not fast .. but first i had 0.0175 and now i have 0.038

**Reply by gasprints** [reply]
I'm referring to the max volume speed.  I lowered it to 12 from 14 and it has helped a little.  Not a perfect fix, quick and dirty.  Keeping my orders going out at least

<img width="393" alt="Screenshot 2023-04-26 at 9 53 50 AM" src="https://user-images.githubusercontent.com/128255447/234647899-91750456-68df-43fa-881c-b25cce680de2.png">

**Reply by Mario-vminion** [reply]
Could a wrong pressure advanced also have small gaps on the top layer i did not change anything 
![Uploading IMG_4618.jpeg…]()
